journalism_workflow/static/journalism_workflow/copy_editor/js/controllers.js

```diff
@@ -0,0 +1,30 @@
+(function () {
+  'use strict';
+
+  angular
+    .module('journalism_workflow.copy_editor.controllers')
+    .controller('CopyEditorController', CopyEditorController)
+
+  CopyEditorController.$inject = ['$scope', 'orchestraService'];
+
+  function CopyEditorController($scope, orchestraService) {
+    var vm = $scope;
+
+    // Store the article text document URL for easier summary later
+    var documentCreationStep = orchestraService.taskUtils.findPrerequisite(
+      vm.taskAssignment, 'document_creation')
+    vm.taskAssignment.task.data.articleDocument = documentCreationStep.task.data.articleURL;
+
+    // Set up the photos for captioning
+    vm.taskAssignment.task.data.photos = [];
+    var photoAdjustStep = orchestraService.taskUtils.findPrerequisite(
+      vm.taskAssignment, 'photo_adjust');
```

Comment:
pro of showing adjusted photos:
- it's pedagogically helpful to understand machine steps---people see that something changed

con:
- it's not true to life, in the sense that we'd realistically be creating a few copies of each image with various color corrections, etc., for different media.  showing a thumbnail of the original image would be more appropriate at that point, which would mean the photo adjustment step would likely want to create a json blob for each image like {'original': url, 'newspaper_thumbnail': url, 'newspaper_hero': url, 'desktop_color': url, ...}